examples/ansible/using_netbox_dns_with_ansible.md

```diff
@@ -199,3 +199,9 @@ test1                                        IN CNAME       test1.zone1.example.
 ```
 
 The new records have been inserted and the zone SOA SERIAL is updated in the SOA record.
+
+#### Creating all Zones organized in Netbox DNS which are active or deprecated
+
+To create all Files to use with Bind DNS, you can use the example stated in Synchronize_DNS_Zones. With that Playbook
+Ansible reads every zone that is active or deprecated (NOT Reserved or Parked) and writes them into a Folder for BIND to read.
+After that the Bind-Service restarts. ATM the Zones are also needed to enter manually into the named.conf.local.
```

Comment:
Tip: If you are not an English native speaker (I am not), running your texts through [DeepL](https://deepl.com) can help with grammar and orthography.